cdap-standalone/src/main/resources/cdap-env.sh

```diff
@@ -0,0 +1,19 @@
+# Copyright © 2018 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Default jvm option for the kill command, it cannot be combined with the SDK options because split_jvm_opts() method will
```

Comment:
This file should have user-friendly comments for each option.  This one is too much technical detail, it should just be something like "By default, kill the CDAP Sandbox process if it runs out of memory".  Also, no "SDK"